src/main/kotlin/utils/MixedTicketGenerationStrategy.kt

```diff
@@ -0,0 +1,33 @@
+package utils
+
+import entity.Ticket
+import model.Amount
+import model.Lottery
+import model.LotteryPurchasePattern
+
+class MixedTicketGenerationStrategy(
+    private val amount: Amount,
+    private val explicitLotteriesInput: List<String>,
+    private val lotteryPurchasePattern: LotteryPurchasePattern,
+) : TicketGenerationStrategy {
+    override fun issueTicket(): Ticket {
+        val lotteries = mutableListOf<Lottery>()
+
+        lotteries.addAll(generateExplicitLotteries())
+        lotteries.addAll(generateRandomLotteries())
+
+        return Ticket(lotteries, amount, lotteryPurchasePattern)
+    }
+
+    private fun generateRandomLotteries(): List<Lottery> {
+        val lotteries = mutableListOf<Lottery>()
+        repeat(lotteryPurchasePattern.autoLottoCount) {
+            lotteries.add(Lottery(RandomLotteryGenerator.generateNumbers()))
+        }
+        return lotteries
+    }
```

Comment:
```suggestion
  private fun generateRandomLotteries() = List(lotteryPurchasePattern.autoLottoCount) {
        Lottery(RandomLotteryGenerator.generateNumbers())
    }
```

가변리스트 대신 불변리스트로도 구현할 수 있다는 것을 참고하시면 좋을 것 같아 의견 남깁니다.

Reply:
이런 방식은 전혀 생각하지 못했습니다. 정말 감사드립니다🔥